src/sentry/integrations/msteams/client.py

```diff
@@ -11,14 +12,15 @@
 from sentry.integrations.services.integration import integration_service
 from sentry.integrations.services.integration.model import RpcIntegration
 from sentry.shared_integrations.client.proxy import IntegrationProxyClient, infer_org_integration
+from sentry.shared_integrations.exceptions import IntegrationError
 from sentry.silo.base import SiloMode, control_silo_function
 
 # five minutes which is industry standard clock skew tolerance
 CLOCK_SKEW = 60 * 5
 
 
-# MsTeamsClientMixin abstract client does not handle setting the base url or auth token
-class MsTeamsClientMixin:
+# MsTeamsClientABC abstract client does not handle setting the base url or auth token
+class MsTeamsClientABC(ApiClient, ABC):
```

Comment:
_ideally_ the mixin would be converted to free functions since it's not really sound (and this patch adds a diamond inheritance) -- but this is an improvement at least